cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -159,12 +161,14 @@ def gen_arg_parser(cls, args_to_parse=None):
                                   help="Provide a ceph conf file.", type=str)
         common_group.add_argument("--run-as-user", "-u", default="", type=str,
                                   help="Provides a user name to check the cluster's health status, must be prefixed by 'client.'")
-        common_group.add_argument("--cluster-name", default="openshift-storage",
+        common_group.add_argument("--cluster-name", default="",
                                   help="Ceph cluster name")
         common_group.add_argument("--namespace", default="",
                                   help="Namespace where CephCluster is running")
         common_group.add_argument("--rgw-pool-prefix", default="",
                                   help="RGW Pool prefix")
+        common_group.add_argument("--restricted-auth-permission",default=False,
```

Comment:
```suggestion
        common_group.add_argument("--restricted-auth-permission", default=False,
```

Do you have a Python linter installed?

Reply:
It's not working still, need to it figure out
But updated the places where it was badly indented.

Reply:
Used Pylint